appveyor.yml

```diff
@@ -36,10 +37,19 @@ install:
   - C:\projects\julia-binary.exe /S /D=C:\projects\julia
 
 build_script:
-# Need to convert from shallow to complete for Pkg.clone to work
-  - IF EXIST .git\shallow (git fetch --unshallow)
-  - C:\projects\julia\bin\julia -e "versioninfo();
-      Pkg.clone(pwd(), \"DataFrames\"); Pkg.build(\"DataFrames\")"
+  - C:\projects\julia\bin\julia -e "
```

Comment:
@simonbyrne, does your new AppVeyor.jl script take this 0.7 version check into account automatically? Should we just switch to that for DataFrames here?

Reply:
Yes it does.